Check for config/initializers/assets.rb file

buren opened the pull request:
First of all thanks for an awesome gem :star: 

react_on_rails:install fails when there is no existing assets initializer file:

```
$ rails generate react_on_rails:install
# ...
# => No such file or directory @ rb_sysopen /Users/xxx../config/initializers/assets.rb
```

I added a small patch that followed the same pattern used in [`BootstrapGenerator#prepend_to_application_scss `](https://github.com/shakacode/react_on_rails/blob/213fd8049840387f32bdb626e20fbde29a5acea5/lib/generators/react_on_rails/bootstrap_generator.rb#L82).

robwise commented:
@buren Would it be possible to also get a test for this? It would basically be just like the application.scss file test here: https://github.com/shakacode/react_on_rails/pull/137

justin808 commented on lib/generators/react_on_rails/bootstrap_generator.rb:
```diff
@@ -20,7 +20,12 @@ def append_to_assets_initializer
+        if File.exist?(assets_intializer)
+          append_to_file(assets_intializer, data)
+        else
+          puts_setup_file_error(assets_intializer, data)
```
we should create the file if it doesn't exist, not error out

robwise replied:
@justin808 actually that's a good point, @buren isn't that what it does for the appliaction.js file?

buren replied:
:+1: I'll update this to use `create_file(assets_intializer, data)` and I'll also add some test.

buren replied:
Thanks @robwise and @justin808 for the quick review! :rocket:

buren commented:
@robwise, @justin808 I replaced `puts_setup_file_error` with `create_file` for `config/initializers/assets.rb` and added test :)

Please let me know if there is anything I can do to improve this PR :+1: 

Thanks!

buren commented:
I also tested/verified this on an app that doesn't have `config/initializers/assets.rb` file.

justin808 commented:
This looks excellent. Once @robwise has a chance to look at it, we'll merge!

justin808 commented:
@buren can you please rebase your commits into one commit on top of master, and then I'll merge. Thanks!

buren commented:
@justin808 rebased and I also added another test for `with assets.rb file`, to follow the same pattern you use with `application.js`..

justin808 commented:
Thanks @buren!

buren commented:
Thank you @justin808, @robwise and others for your hard work with this project! :+1:
